src/db/models/task.js

```diff
@@ -0,0 +1,52 @@
+import {Model} from "sequelize"
+
+export default (sequelize, DataTypes) => {
+  class Task extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Task.belongsTo(models.User, {
+        foreignKey: 'userId',
+        onDelete: 'CASCADE'
+      })
+      Task.hasOne(models.User, {foreignKey: 'userId'})
```

Comment:
@lateefat-taiwo can you explain why you defined this and the following line association?